test/common/upstream/load_balancer_benchmark.cc

```diff
@@ -79,10 +136,19 @@ void BM_RingHashLoadBalancerBuildRing(benchmark::State& state) {
     const uint64_t num_hosts = state.range(0);
     const uint64_t min_ring_size = state.range(1);
     RingHashTester tester(num_hosts, min_ring_size);
-    state.ResumeTiming();
```

Comment:
why these changes to the ring hash part of this code? is this expected to change the behavior of the consistent hashing LBs as well?

Reply:
Remove changes to benchmarks for other LBs.